base_optional_quick_create/model.py

```diff
@@ -38,16 +39,42 @@ def wrapper(cr, uid, name, context=None):
         return wrapper
 
     def _register_hook(self, cr, ids=None):
```

Comment:
Just cooked up this (untested): https://github.com/OCA/server-tools/pull/340/files